numba/targets/arrayobj.py

```diff
@@ -1838,6 +1838,29 @@ def np_unique_impl(a):
     return np_unique_impl
 
 
+@overload(np.repeat)
+def np_repeat(a):
+    def np_repeat_impl(a, repeats, axis=None):
+        # a, repeats, axis = a_input
+        to_return = []
```

Comment:
Am I right in thinking that the array size for the output array is a direct function of `a` and `repeats` ? If so, to make this efficient, could the output array be allocated upfront and then the repeated values assigned into that array?